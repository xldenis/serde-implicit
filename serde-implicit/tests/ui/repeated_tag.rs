#[derive(serde_implicit_proc::Deserialize)]
enum RepeatedTag {
    Var1 {
        #[serde_implicit(tag)]
        primary_tag: String,
        value: u32,
    },
    Var2 {
        #[serde_implicit(tag)]
        primary_tag: String,
        value: String,
    },
}

fn main() {}
