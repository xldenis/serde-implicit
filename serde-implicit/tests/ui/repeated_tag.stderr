error: duplicate tags found
  --> tests/ui/repeated_tag.rs:10:9
   |
10 |         primary_tag: String,
   |         ^^^^^^^^^^^
